### PR TITLE
Simple benchmark application

### PR DESCRIPTION
**Pull request by penberg**
This adds a very simple application for benchmarking purposes. The main
benefit right now is the fixture setup using Faker.js, but we can keep
on improving this with more scenarios and more automation for comparing
performance.

### CODE REVIEW

File: perf/backend/endpoints/hello.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default async function (req: Request): Promise<Response> {
```
Comment (dekimir) [inline perf/backend/endpoints/hello.ts:1]: Is this one actually used?  If not, let's not check it in.

**Review by dekimir — CHANGES_REQUESTED** [review]
Looks good, but drop hello.ts.

File: perf/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Start ChiselStrike server:
+
+```console
+cd backend && ../../target/debug/chisel dev
+```
+
+Generate some test data:
+
+```console
+for i in {1..1000}; do curl -X POST localhost:8080/dev/fixture; done
```
Comment (dekimir) [inline perf/README.md:14]: Doesn't fixture already have its own loop?

Comment (penberg) [reply perf/README.md]: @dekimir Yeah, fixed the README now.

**Review by dekimir — APPROVED** [review]

**Pull request merged**